core/src/main/java/org/mobilitydata/gtfsvalidator/table/AnyTableLoader.java

```diff
@@ -21,13 +22,27 @@
 import org.mobilitydata.gtfsvalidator.parsing.CsvRow;
 import org.mobilitydata.gtfsvalidator.parsing.FieldCache;
 import org.mobilitydata.gtfsvalidator.parsing.RowParser;
+import org.mobilitydata.gtfsvalidator.validator.FileValidator;
 import org.mobilitydata.gtfsvalidator.validator.SingleEntityValidator;
 import org.mobilitydata.gtfsvalidator.validator.ValidatorProvider;
 import org.mobilitydata.gtfsvalidator.validator.ValidatorUtil;
 
 public final class AnyTableLoader {
 
   private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+  private static final List<Class<? extends FileValidator>> validatorsWithParsingErrors =
+      new ArrayList<>();
```

Comment:
Also it might be a good idea to rename the `skippedValidators` list used in existing code to `multiFileValidatorsWithParsingErrors`. Your way of naming is more descriptive (`skippedValidators` does not tell us why it was skipped, whereas `singleEntityValidatorsWithParsingErrors` for example does)
This change would not be in your code, so I realize it's a bit more touchy.

Reply:
Renaming Done.